### PR TITLE
Feature/upgrade logging v2

### PR DESCRIPTION
**Pull request by stmuk**
### What

Upgrade to logging v2

### How to review

Run tests & look at diff

### Who can review

Anyone

### CODE REVIEW

File: main_test.go [hunk 1]
```diff
@@ -31,7 +32,7 @@ func (f *ComponentTest) InitializeScenario(ctx *godog.ScenarioContext) {
 
 	ctx.AfterScenario(func(*godog.Scenario, error) {
 		if err = component.Close(); err != nil {
-			log.Event(nil, "error closing identity component", log.Error(err), log.WARN)
+			log.Warn(context.Background(), "error closing identity component", log.FormatErrors([]error{err}))
```
Comment (redhug1) [inline main_test.go:35]: What is the reason for using:
log.FormatErrors([]error{err}
instead of just:
err
?

Comment (stmuk) [reply main_test.go:35]: I think it's an error caused by the automatic process I'm using.

I'll simplify it.

Comment (stmuk) [reply main_test.go:35]: The func signature is Warn(ctx context.Context, event string, opts ...option) and doesn't contain an error type.

Comment (redhug1) [reply main_test.go:35]: Ok.Thanks.

**Review by redhug1 — APPROVED** [review]
LGTM

**Pull request merged**